### PR TITLE
Minor updates to standard/linq

### PR DESCRIPTION
**Pull request by Youssef1313**

### CODE REVIEW

File: docs/standard/linq/atomized-xname-xnamespace-objects.md [hunk 1]
```diff
@@ -91,9 +91,9 @@ foreach (var z in query)
 ```vb
 Dim root As New XElement("Root", New XElement("C1", 1), New XElement("Z1", New XElement("C1", 2), New XElement("C1", 1)))
 
-Dim query = From e In root.Descendants("C1") Where CInt(e) = 1e
+Dim query = From e In root.Descendants("C1") Where CInt(e) = 1
 
-For Each z As var In query
+For Each z In query
```
Comment (Youssef1313) [inline docs/standard/linq/atomized-xname-xnamespace-objects.md:96]: These are compile-errors

File: docs/standard/linq/add-elements-attributes-nodes-xml-tree.md [hunk 2]
```diff
@@ -80,7 +80,7 @@ Dim xmlTree As XElement = _
 xmlTree.Add(<NewChild>new content</NewChild>)
 xmlTree.Add( _
     From el In srcTree.Elements() _
-    Where CInt(el) > 3 _
+    Where CInt(el) > 3
```
Comment (gewarren) [inline docs/standard/linq/add-elements-attributes-nodes-xml-tree.md:83]: Why was this one removed by not the one on lines 81 and 82?

Comment (Youssef1313) [reply docs/standard/linq/add-elements-attributes-nodes-xml-tree.md:83]: They were forgotten 😄 
Thanks for catching, will update.

File: docs/standard/linq/add-elements-attributes-nodes-xml-tree.md [hunk 3]
```diff
@@ -80,7 +80,7 @@ Dim xmlTree As XElement = _
 xmlTree.Add(<NewChild>new content</NewChild>)
 xmlTree.Add( _
     From el In srcTree.Elements() _
```
Comment (Youssef1313) [inline docs/standard/linq/add-elements-attributes-nodes-xml-tree.md:82]:
```suggestion
xmlTree.Add(
    From el In srcTree.Elements()
```

**Review by gewarren — APPROVED** [review]

**Pull request merged**